app/src/main/java/com/hms/explorehms/onboarding/adapter/OnboardingAdapter.java

```diff
@@ -45,9 +45,9 @@ public Fragment createFragment(int position) {
         } else if (position == 1) {
```

Comment:
if you need to write more than 3 condition, you should to think use switch case

Reply:
I didn't set the logic but I fixed it